FusionMCPBridge/.env

```
@@ -0,0 +1 @@
+PYTHONPATH=/Users/bsport/Library/Application Support/Autodesk/webdeploy/production/7358a48d390859c49ebd8e305c032963f33f9277/Api/Python/packages
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)